datasafe-long-run-tests/datasafe-business-tests-random-actions/parse_script_modif.groovy

```diff
@@ -0,0 +1,50 @@
+import java.nio.file.Files
```

Comment:
It would be good to have some reference in documentation for this script and to explain what it is doing

Reply:
ok